pages/places/index.js

```diff
@@ -30,6 +30,10 @@ module.exports = ({ content } = {}) => {
         comparator: 'AND',
         exact: true
       },
+      notes: {
+        show: true,
```

Comment:
I don't think this should be `show`n - it's appeared in the main body of the table.

Reply:
Also, does `expandable` have any meaning now? 👇

Reply:
Looks like "no", so I've removed this.